py3status/modules/yandexdisk_status.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Display Yandex.Disk status.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (10s default)
+    format: prefix text for the Yandex.Disk status
+
+Valid status values include:
+    - Yandex.Disk isn't running!
+    - Error: daemon not started
+    - ...
```

Comment:
@tobes
But I have it. Repo was cloned (`git clone URL`). I've made a build (`python ./setup.py clean build`), installed it (`sudo python ./setup.py install`). After that I've restarted my i3. At this point I have the py3status fresh installed built from repo. And then I get this error above.
Actually I thought that the `docstring update` command will work as the Doxygen works and documentation updates in current development directory, so it's strange to see that `docstring update` tries to find somethin in already installed software at /usr/local/dir.
I'll try to find out how it can be done, it's just looks unusual.